smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiConfig.java

```diff
@@ -48,6 +48,16 @@ public enum HttpPrefixHeadersStrategy {
         WARN
     }
 
+    /** Specifies how to resolve multiple error responses with same error codes. */
+    public enum ErrorStatusConflictHandlingStrategy {
+        /** the default setting that uses OpenAPI's oneOf keyword to combine multiple schemas for same media type. */
+        USE_ONEOF,
+        /** The custom setting that combines multiple schemas for same media type using properties field of
```

Comment:
nit - expand this javadoc to use the normal form since it doesn't fit on a single line:
```
/**
 * The custom ....
 */
```

Reply:
resolved.